andela/phlox-ah-frontend
# #160451474 user should be able to bookmark article

What does this PR do?
This PR adds the functionality to bookmark an article.

Description of Task to be completed?
Bookmark an article
#### How should this be manually tested?
Login and go to an article that doesn't belong to you.
On clicking the bookmark icon, the article should be bookmarked
#### Any background context you want to provide?
None
### Screenshots
None
### What are the relevant pivotal tracker stories?
#160451474
